src/applications/ivc-champva/shared/components/fileUploads/MissingFileList.jsx

```diff
@@ -80,24 +75,40 @@ export default function MissingFileList({
   return (
     <div>
       <h3 className="vads-u-font-size--h4">{title || ''}</h3>
-      <p>{description || ''}</p>
+      <p>
+        <strong>{description || ''}</strong>
+      </p>
       {wrapped.map((entry, idx) => {
-        if (entry?.missingUploads.filter(f => inSubset(f)).length === 0)
+        if (
+          entry?.missingUploads.filter(f => f.required === subset).length === 0
+        )
           return <></>;
         const entryName = `${entry[nameKey].first} ${entry[nameKey]?.middle ||
           ''} ${entry[nameKey].last}${entry[nameKey]?.suffix || ''}`;
         return (
           <div key={`${entryName}-${subset}`}>
-            <strong>{entryName}</strong>
-            <ul style={!disableLinks ? { listStyleType: 'none' } : {}}>
+            {showNameHeader ? <strong>{entryName}</strong> : null}
+            <ul
+              style={
+                !disableLinks && !showFileBullets
+                  ? { listStyleType: 'none' }
+                  : { listStylePosition: 'inside' }
+              }
+            >
               {entry.missingUploads?.map((file, index) => {
-                return inSubset(file) &&
+                const fn =
+                  fileNameMap?.[file.name] ?? makeHumanReadable(file.name);
+                return file.required === subset &&
                   (disableLinks ? file.uploaded === false : true) ? (
                   <li key={file.name + file.uploaded + index}>
-                    {!disableLinks
-                      ? alertOrLink(file, entryName, idx, fileNameMap)
-                      : fileNameMap?.[file.name] ??
-                        makeHumanReadable(file.name)}
+                    {!disableLinks ? (
+                      <>
+                        {showFileBullets ? fn : null}
+                        {alertOrLink(file, entryName, idx, fileNameMap)}
+                      </>
+                    ) : (
+                      fn
+                    )}
```

Comment:
Updated this so that it now conditionally renders either:
1. A bulleted list of just text
2. A non-bulleted list with custom file upload links
3. A non-bulleted list with custom file upload success messages
4. [2] and [3] but with an additional textual bullet directly above the link/success message